src/view/FilesAppIntegration.js

```diff
@@ -91,10 +91,8 @@ export default {
 			return
 		}
 
-		if (this.getFileList()) {
-			this.getFileList().setViewerMode && this.getFileList().setViewerMode(false)
-			this.getFileList().reload && this.getFileList().reload()
-		}
+		this.updateFileInfo(undefined, Math.floor(Date.now()))
```

Comment:
How about `Date.now()` instead of `1Math.floor(Date.now())`? They return same value (integer).